src/macos-ip_mroute.h

```diff
@@ -0,0 +1,293 @@
+/*
+ * Copyright (c) 2000 Apple Computer, Inc. All rights reserved.
+ *
+ * @APPLE_LICENSE_HEADER_START@
+ * 
+ * The contents of this file constitute Original Code as defined in and
+ * are subject to the Apple Public Source License Version 1.1 (the
+ * "License").  You may not use this file except in compliance with the
+ * License.  Please obtain a copy of the License at
+ * http://www.apple.com/publicsource and read it before using this file.
```

Comment:
Please read license properly prior using it.

**You may not use this file except in compliance with the License.**

This Apple Public Source License Version 1.1 is not compatible with GPLv2 or later license (under which is igmpproxy) and therefore you cannot use this file in igmpproxy.

It means that including this change into igmpproxy is violation of license arguments of copyright holder of this file and therefore changes in this pull requests cannot be used or included into igmpproxy project.

Apple Public Source License Version 1.1 is moreover [non-free license](https://www.gnu.org/philosophy/historical-apsl.html) and cannot be used in any free software (not only in igmpproxy).

Reply:
Thank you for the response!

Sorry, didn't really dig info about this license... Guess I'll have to close the PR.